docs/typescript.md

```diff
@@ -3,33 +3,33 @@ id: typescript
 title: Using TypeScript with React Native
 ---
 
-[TypeScript][ts] is a language which extends JavaScript to add type definitions, a lot like [Flow][flow] which React Native is built in. React Native supports TypeScript and Flow out of the box by default.
+[TypeScript][ts] is a language which extends JavaScript to add type definitions, much like [Flow][flow]. While React Native is built in Flow, it supports both TypeScript _and_ Flow by default.
```

Comment:
Seems to be more confusing than what was previously there.

`React Native supports TypeScript and Flow out of the box by default.` was clearer

Reply:
"[TypeScript][ts] is a language which extends JavaScript to add type definitions, much like [Flow][flow]. React Native is built in Flow, and supports both TypeScript _and_ Flow by default."